tsdb/engine/tsm1/cache_test.go

```diff
@@ -223,7 +225,7 @@ func TestCacheLoader_LoadSingle(t *testing.T) {
 		Values: values,
 	}
 
-	if err := w.Write(entry); err != nil {
+	if err := w.Write(mustMarshalEntry(entry)); err != nil {
```

Comment:
No, I refactored the write method to take the marshaled entry (type and compressed bytes). That's part of my whole write lock cleanup